tests/integration-tests/tests/update/test_update.py

```diff
@@ -749,3 +761,295 @@ def _test_update_queue_strategy_with_running_job(
     _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
     _check_queue_ami(cluster, ec2, pcluster_copy_ami_id, "queue2")
     assert_compute_node_states(scheduler_commands, queue1_nodes, "idle")
+
+
+@pytest.mark.usefixtures("instance")
+def test_dynamic_file_systems_update(
```

Comment:
Yes, it make sense to put it in test/storages. I think it also make sense put it under tests/update because it only related to update of FS and related to queue_update_stragegy.
Both queue_update_strategy and update functionality are under test_update

Reply:
Not a blocking comment, but I still think it should be within the storage folder.